docs/docsite/rst/shared_snippets/installing_collections_git_repo.txt

```diff
@@ -13,29 +18,26 @@ You can install a collection in a git repository by providing the URI to the rep
    ansible-galaxy collection install git@github.com:organization/repo_name.git
 
    # Install a collection from a local git repository
-   ansible-galaxy collection install git+file:///home/user/path/to/repo/.git
-
-In a ``requirements.yml`` file, you can also use the ``type`` and ``version`` keys in addition to using the ``git+repo,version`` syntax for the collection name.
-
-.. code-block:: yaml
-
-   collections:
-     - name: https://github.com/organization/repo_name.git
-       type: git
-       version: devel
+   ansible-galaxy collection install git+file:///home/user/path/to/repo_name.git
 
-Git repositories can be used for collection dependencies as well. This can be helpful for local development and testing but built/published artifacts should only have dependencies on other artifacts.
+.. warning::
 
-.. code-block:: yaml
+   Embedding credentials into a git URI is not secure. Use safe authentication options to prevent your credentials from being exposed in logs or elsewhere. Safe authentication options include:
```

Comment:
```suggestion
   Embedding credentials into a git URI is not secure. Use safe authentication options to prevent your credentials from being exposed in logs or elsewhere. 
```
You are using full sentences for the bullet points. You don't need to say "safe authentication options include:". Add a period at the end of each bullet point.

Reply:
Good point.